fth/file.fth

```diff
@@ -50,6 +50,8 @@ private{
 create (LINE-TERMINATOR) \n c,
 : LINE-TERMINATOR ( -- c-addr u ) (line-terminator) 1 ;
 
+-72 constant THROW_RENAME_FILE
```

Comment:
Are these numbers, eg. -72,  part of a standard?
If so please document, or let's define them all in one place.